File: conditionals/time_of_day.c
// Anthony Petersen, Template
#include <stdio.h>
#include <time.h>



int main(void){
    time_t now = time(NULL);
    struct tm * tm_struct = localtime(&now);
    int hour = tm_struct->tm_hour;
    if(hour >= 17){
        printf("Good evening! \n");
    }else if(hour >= 12){
        printf("Good afternoon! \n");
    }else
        printf("Good morning!");
    return 0;
}

File: Conditionals/time_of_day.py
#Anthony Petersen, Time Of Day
import time
current = time.time()
local_time = time.localtime(current)
hour = local_time.tm_hour
if(hour >= 17):
    print("Good evening! \n")
elif(hour >= 12):
    print("Good afternoon! \n")
else:
    print("Good morning!")